package.json

```diff
@@ -145,16 +145,35 @@
     "mime-types": "^3.0.1",
     "srcset": "^5.0.2"
   },
+  "storybook": {
+    "displayName": "Happo",
+    "icon": "https://happo.io/static/happo-hippo.png",
```

Comment:
On closer inspection, I'm not sure this is a good image for this. It is not very crisp and it seems to have some background texture of some sort.